internal/cmd/skupper/token/kube/token_issue.go

```diff
@@ -70,9 +71,15 @@ func (cmd *CmdTokenIssue) ValidateInput(args []string) error {
 		if !ok {
 			validationErrors = append(validationErrors, fmt.Errorf("token file name is not valid: %s", err))
 		} else {
-			// check we can use as a filename
-			if _, err := os.ReadDir(args[0]); err == nil {
+			directory, filename := filepath.Split(args[0])
+			if filename == "" {
+				validationErrors = append(validationErrors, fmt.Errorf("token file name is not valid"))
+			} else if directory == "/" {
```

Comment:
As long as the filename is not empty (which is the case here), I believe that `/` can be considered as a valid directory.

Maybe a more generic validation would be to use, like, `os.CreateTemp(directory, "skupper-token.*")` to create a temporary file (removing it later) to make sure the current user has permission to write into `directory`, before proceeding with the token generation procedure.